go.mod

```diff
@@ -42,7 +41,7 @@ require (
 	github.com/caarlos0/env/v6 v6.10.1
 	github.com/gobwas/glob v0.2.3
 	github.com/google/go-github/v53 v53.2.0
-	github.com/google/osv-scanner v1.9.0
+	github.com/google/osv-scanner v1.9.2
```

Comment:
this upgrade should be fine performance wise, but I'd like to wait until after the holidays in case it needs to be fixed.